proto/kuksa/val/v2/types.proto

```diff
@@ -128,20 +128,20 @@ message ValueRestriction {
 }
 
 message ValueRestrictionInt {
-  optional sint64 min            = 1;
-  optional sint64 max            = 2;
+  sint64 min            = 1;
```

Comment:
Maybe add some generic documentation/comment on that the values reported are the actual limits checked/enforced by Databroker, which may be either values specified in VSS, or if not present implicit limits for the datatype.

Reply:
Added